CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
 - Bump CoreDNS version to 1.8.0
   (PR[#3354](https://github.com/scality/metalk8s/pull/3354))
 
+### Deprecations and Removals
+
+- [#3357](https://github.com/scality/metalk8s/issues/3357) - Drop Ubuntu support
+  (PR [#3358](https://github.com/scality/metalk8s/pull/3358))
+
```

Comment:
TBH, we never had Ubuntu support, nor advertised it. So we may want to log it here (debatable, IMO), but I wouldn't say we're dropping support, more like "Stop Ubuntu support development due to lack of interest", or something along those lines. WDYT @NicolasT?

Reply:
:+1: to me we should either do not mention it all in the changelog, either we just say "remove dead code" (or as Guillaume said stop dev)

Reply:
Since we have never documented anything on Ubuntu support (and there is nothing in changlogs either), I figured no changelog entry at all would be fine.

Reply:
I've removed the changelog entry.